frontend/src/components/TaskList.js

```diff
@@ -158,7 +158,7 @@ TaskList.propTypes = {
 };
 
 const mapStateToProps = state => ({
-    tasks: state.tasks
+    tasks: state.tasks.tasks
```

Comment:
state.tasks.tasks => looks weird

Reply:
check that with Ewelina, maybe it should be like that .. I don't know, but looks weird ;)

Reply:
```
const { tasks } = state.tasks

return { tasks }
```

you will usually return more than one variable from state so I think it's cleaner that way

you could also name name reducer `tasksReducer` instead of `tasks` if currently it's confusing

Reply:
done